src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+import "./avatar.css";
+import AVATAR_LIST from "../../avatarList";
+import { AvatarId } from "types/avatars";
+
+type Props = {
+  id: AvatarId;
+};
+
+const Avatar: React.FunctionComponent<Props> = ({ id }) => {
+  const avatar = AVATAR_LIST.find((avatar) => avatar.id === id);
+
+  return (
+    <img
+      className="avatar-image"
+      data-testid="avatar"
+      src={avatar?.src}
```

Comment:
I would probably `return null` early here if no `avatar` was found above.  Not much sense in rendering an `img` if there is nothing found.